db/models.py

```diff
@@ -6,6 +6,7 @@ class User(models.Model):
     last_name = models.CharField(max_length=63)
     username = models.CharField(max_length=63, unique=True)
     bio = models.CharField(max_length=255)
+    num_messages = models.IntegerField(null=True)
```

Comment:
I'm not sure if it's okay to change models here. Either way, it's not necessary to implement `get_last_5_messages_dicts()`.